### PR TITLE
Add release notes for web-poet 0.7.0

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

**Comment by Gallaecio** [comment]
The test failure is caused by https://github.com/tox-dev/tox/issues/2817

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+0.6.1 (to be released)
+----------------------
+
+* Support page object serialization, to make it easier to implement caching
+  mechanisms in web-poet frameworks.
```
Comment (kmike) [inline CHANGELOG.rst:9]: I think we want to release it only after https://github.com/scrapinghub/web-poet/pull/110 is merged. 

* Serialization is not that useful on its own; 
* we're not switching scrapy-poet cache to it yet (right @wRAR?);
* there are some significant changes in the testing PR related to serialization.

**Comment by wRAR** [comment]
> The test failure is caused by [tox-dev/tox#2817](https://github.com/tox-dev/tox/issues/2817)

https://github.com/scrapinghub/web-poet/pull/110/commits/753621579e45e9ebc020de3f21e749c898b2dcbe should fix this.

**Comment by wRAR** [comment]
I didn't cover #114, feel free to proposea line for it if you want ("CI improvements"?).

**Comment by wRAR** [comment]
https://community.codecov.com/t/upload-issues-unable-to-locate-build-via-github-actions-api/3954

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,8 +2,15 @@
 Changelog
 =========
 
-TBR
----
+0.7.0 (to be released)
+----------------------
+
+* Support page object dependency serialization, to make it easier to implement
+  caching mechanisms in web-poet frameworks.
```
Comment (BurnzZ) [inline CHANGELOG.rst:9]: I think it'd be great to add a link to the new doc that we have on this.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,8 +2,15 @@
...
+
```
Comment (BurnzZ) [inline CHANGELOG.rst:10]: Should we also mention the new deps in `install_requires`?

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,8 +2,16 @@
 Changelog
 =========
 
-TBR
----
+0.7.0 (to be released)
+----------------------
+
+* Support :ref:`page object dependency serialization <dep-serialization>`, to
+  make it easier to implement caching mechanisms in web-poet frameworks.
```
Comment (kmike) [inline CHANGELOG.rst:9]: Is it the main use case for serialization?

Comment (wRAR) [reply CHANGELOG.rst:9]: This line was added before the testing framework was merged. Do you think we need to keep it at all as it's a part of the testing feature?

Comment (Gallaecio) [reply CHANGELOG.rst]: What about https://github.com/scrapinghub/web-poet/pull/111/commits/f0657ea820780786d5e73503fefc640966bbcda8 ?

**Review by kmike — APPROVED** [review]
Looks good!

File: CHANGELOG.rst [hunk 5]
```diff
@@ -17,6 +22,13 @@ TBR
   ``instead_of`` or ``to_return`` present in multi-parameter search calls would
   filter the initial candidate results resulting in a faster search.
 
+* Support :ref:`page object dependency serialization <dep-serialization>`.
+
+* Add new dependencies used in testing and serialization code: ``andi``,
+  ``python-dateutil`` and ``time-machine``. Also ``backports.zoneinfo`` on
```
Comment (BurnzZ) [inline CHANGELOG.rst:28]:
```suggestion
  ``python-dateutil``, and ``time-machine``. Also ``backports.zoneinfo`` on
```

minor: Oxford comma

**Review by BurnzZ — APPROVED** [review]

**Pull request merged**